packages/amazonq/test/e2e/amazonq/framework/framework.ts

```diff
@@ -29,7 +29,7 @@ export class qTestingFramework {
         featureName: TabType,
         amazonQEnabled: boolean,
         featureConfigsSerialized: [string, FeatureContext][],
-        welcomeCount = 0
+        welcomeCount = 10 // by default don't show the welcome page
```

Comment:
If this number is hardcoded in the prod code, might be better to export a common constant